pkg/utils/cmd_exec_test.go

```diff
@@ -0,0 +1,51 @@
+// +build darwin
```

Comment:
Any specific reason for including this in `darwin` build ? Am I missing something ?

Reply:
This is just a miss, I initially wanted to have it only for mac because os some reason., lemme remove this.